search/processingmodules.md

```diff
@@ -79,6 +79,7 @@ subnet <subnet/subnet>
 taint <taint/taint>
 time <time/time>
 transaction <transaction/transaction>
+trim <trim/trim>
```

Comment:
I would recommend that we make this `Trim Modules (list)` here the same way that we do Math Modules. This is particularly relevant if users are navigating using the left hand menu. It lets them know that there are more trim options listed on the page.

Reply:
Done